lib/block_view_types.go

```diff
@@ -98,8 +102,10 @@ const (
 	OperationTypeAcceptNFTTransfer            OperationType = 21
 	OperationTypeBurnNFT                      OperationType = 22
 	OperationTypeAuthorizeDerivedKey          OperationType = 23
+	OperationTypeMessagingKey                 OperationType = 24
+	OperationTypeMessageParty                 OperationType = 25
 
-	// NEXT_TAG = 24
+	// NEXT_TAG = 26
```

Comment:
merge main into your branch please - want to be sure we don't have overlap here.